src/main/java/com/fibermc/essentialcommands/mixin/TeleportCommandMixin.java

```diff
@@ -35,9 +35,8 @@ private static void execute(
         @Coerce Object facingLocation,
         CallbackInfo ci
     ) throws CommandSyntaxException {
-        if (target instanceof ServerPlayerEntity) {
+        if (target instanceof ServerPlayerEntity targetPlayer) {
```

Comment:
Evidently I am living in the past. Did not know `instanceof` could be a pattern-matching declaration operator-thing. Cool!

Reply:
IntelliJ starts complaining when I don't do it for some reason